package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hash-for-dep",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
This would probably be a major version change. Nevertheless, the version probably shouldn't be touched in this PR.